api/v1/loadtest_types.go

```diff
@@ -320,7 +320,7 @@ type LoadTestSpec struct {
 // components. If any one component has errored, the load test will be marked in
 // an Errored state, too. This will occur even if the other components are
 // running or succeeded.
-// +kubebuilder:default=Unrecognized
+// +kubebuilder:default=Unknown
```

Comment:
Could you provide reference for usage of `+kubebuilder:default=`, I want to learn more about markers however could not find example of using `+kubebuilder:default=`. Thanks.

Reply:
The `+kubebuilder:default=` marker has no examples. The documentation is virtually nothing more than its signature: https://kubebuilder.io/reference/markers/crd-validation.html#:~:text=default. Defaults will not work unless we upgrade to Kubebuilder v3. We need a bug to do that anyway.

Reply:
Thank you for pointing the reference, wondering what is the difference for Unrecognized and unknow?